### PR TITLE
Add Travis CI build

### PR DESCRIPTION
**Pull request by caniszczyk**
Signed-off-by: Chris Aniszczyk <caniszczyk@gmail.com>

### CODE REVIEW

**Comment by marcosnils** [comment]
@caniszczyk runC already has a janky CI. Is there a specific reason to use Travis?

**Comment by caniszczyk** [comment]
@marcosnils ah interesting, where are the latest builds then? this doesn't look right: https://jenkins.dockerproject.org/job/runc%20Master/

**Comment by cyphar** [comment]
I'm :+1: on using Travis. Currently the Janky stuff is hosted by Docker which means that other maintainers (everyone except @crosbymichael) have very little control about the CI (we can't even see the script that gets run on each build).

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+install:
+  - wget https://github.com/seccomp/libseccomp/archive/v2.3.1.tar.gz
+  - tar -xzvf v2.3.1.tar.gz
+  - pushd libseccomp-2.3.1 && ./configure --prefix=/usr && make && sudo make install && popd
+
+addons:
+  apt:
+    sources:
+    - lucid
```
Comment (cyphar) [inline .travis.yml]: I'm certain that Ubuntu 10.04 doesn't have `bats`.

Comment (caniszczyk) [inline .travis.yml]: thanks @cyphar, just trying to figure how to get this working on the Travis CI containerized infra w/o building everything from source :)

Comment (cyphar) [inline .travis.yml]: /me will play around with it and figure out what will work.

**Pull request closed without merging**